### PR TITLE
Strikes Update

### PR DESCRIPTION
**Pull request by mikemayhemdev**
All Strike Pack cards are now attacks that deal 6 damage!

Also a couple minor downfall pack tweaks:

Ghostflame Strike - Upgrade is now just +3 damage.  Keyword updated for Ghostflame orb.

Shaper's Blessing - minus 1 base block.

### CODE REVIEW

File: src/main/java/thePackmaster/cardmodifiers/strikepack/AddStrikeTagModifier.java [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
     }
 
     @Override
     public void onInitialApplication(AbstractCard card) {
-        super.onInitialApplication(card);
-        if (!card.hasTag(AbstractCard.CardTags.STRIKE)){
-            card.tags.add(AbstractCard.CardTags.STRIKE);
-            card.applyPowers();
-        }
+        card.tags.add(AbstractCard.CardTags.STRIKE);
```
Comment (erasels) [inline src/main/java/thePackmaster/cardmodifiers/strikepack/AddStrikeTagModifier.java:20]: Why remove the check to not add a duplicate tag?

File: src/main/java/thePackmaster/powers/strikepack/StrikeOfGeniusPower.java [hunk 2]
```diff
@@ -21,14 +31,20 @@ public StrikeOfGeniusPower(AbstractCreature owner, int amount) {
...
+        if (amount > 0) {
+            Wiz.atb(new StrikeOfGeniusAction(target));
+            amount--;
+        }
+
+        //amount change is happening instantly in the event of many attacks on the stack simultaneously.
+        if (amount <= 0) {
+            removeThis();
         }
+        super.onAttack(info, damageAmount, target);
```
Comment (erasels) [inline src/main/java/thePackmaster/powers/strikepack/StrikeOfGeniusPower.java:47]: staph

File: src/main/resources/anniv5Resources/localization/eng/strikespack/Cardstrings.json [hunk 3]
```diff
@@ -9,35 +9,36 @@
...
   "${ModID}:StrikeOfGenius": {
     "NAME": "Strike of Genius",
-    "DESCRIPTION": "At the start of your turn, gain a random \"Strike\". NL It costs 0 until played and gains Exhaust.",
-    "UPGRADE_DESCRIPTION": "At the start of your turn, gain a random Upgraded \"Strike\". NL It costs 0 until played and gains Exhaust."
+    "DESCRIPTION": "Deal !D! damage. NL The next attack you play gains 'Gain Block equal to unblocked damage dealt'."
   },
   "${ModID}:StrikeABargain": {
     "NAME": "Strike a Bargain",
-    "DESCRIPTION": "Reduce the cost of Attacks in hand to 0 this turn. You can only play Attacks this turn."
+    "DESCRIPTION": "Deal !D! damage. NL Reduce the cost of Attacks in hand to 0 this turn. NL Exhaust.",
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/strikespack/Cardstrings.json:41]: Untile played, right?

**Review by erasels — CHANGES_REQUESTED** [review]
Also please update the zhs strings as well (just replace them with the english text)

File: src/main/java/thePackmaster/cardmodifiers/strikepack/AddStrikeTagModifier.java (hunk 1 above)
Comment (mikemayhemdev) [reply src/main/java/thePackmaster/cardmodifiers/strikepack/AddStrikeTagModifier.java:20]: The card itself already does the check, so this was an unnecessary double check.

**Comment by mikemayhemdev** [comment]
Alright, this should be ready to roll again.

**Pull request merged**